spec/unit/lib/cloud_controller/runner_spec.rb

```diff
@@ -120,17 +158,15 @@ module VCAP::CloudController
         allow(StenoConfigurer).to receive(:new).and_return(steno_configurer)
         allow(steno_configurer).to receive(:configure)
 
-        subject.run!
-        subject.run!
+        subject
```

Comment:
The test is called `only sets up logging once` and there were two invocations of `subject.run!`. So I guess there should be two invocations of `subject` now.